### PR TITLE
add lr setting description

### PR DESCRIPTION
**Pull request by lm-konda**
Object Detection has already tuned learning settings.  
So, It is only updating doc so far.

And optimizer selector also is not included because it's not merged yet.

### CODE REVIEW

**Comment by ruimashita** [comment]
@lm-konda 
Thanks :+1: 
Current object detection's template parameters of learning rate fit only for PASCAL VOC dataset. It is not good for blue-oil user because they use various datasets.

Please change also object detection's template same as like classification's.
https://github.com/blue-oil/blueoil/blob/master/blueoil/templates/lmnet/object_detection.tpl.py#L101-L109

**Comment by lm-konda** [comment]
Object Detection cannot be applied to template.
Because original template include description like this. How shold we handle this?
```
NETWORK.LEARNING_RATE_KWARGS = {
        "values": [5e-4, 2e-2, 5e-3, 5e-4],
        "boundaries": [step_per_epoch, step_per_epoch * 80, step_per_epoch * 120],
}
```

**Comment by ruimashita** [comment]
@lm-konda 
Why can't you change object detection template ?
Some error comes ?
The object detection learning rate parameters fit only for PASCAL VOC dataset, We want more general one, Maybe you don't need to care to change the template.

**Comment by lm-konda** [comment]
I applied lr setting to object detection.
But It seems to be zero 'boundaries' when train dataset number is not enough.
How do we manage this?

**Comment by ruimashita** [comment]
@lm-konda 
Does zero `boundaries` raise error ?

**Comment by lm-konda** [comment]
I don't think that 'boundaries' itself causes an error.
I guess 'float(dataset_obj.num_per_epoch)/BATCH_SIZE' might be zero.

**Comment by ruimashita** [comment]
related issue: #3

**Comment by ruimashita** [comment]
@hadusam 
dlk jenkins raise error.
```
Receiving objects:  18% (258/1388), 109.12 MiB | 189.00 KiB/s   
Receiving objects:  18% (258/1388), 109.50 MiB | 236.00 KiB/s   
Receiving objects:  18% (258/1388), 109.65 MiB | 253.00 KiB/s   
fatal: The remote end hung up unexpectedly
fatal: early EOF
fatal: index-pack failed

	at org.jenkinsci.plugins.gitclient.CliGitAPIImpl.launchCommandIn(CliGitAPIImpl.java:2016)
	at org.jenkinsci.plugins.gitclient.CliGitAPIImpl.launchCommandWithCredentials(CliGitAPIImpl.java:1735)
	at org.jenkinsci.plugins.gitclient.CliGitAPIImpl.access$300(CliGitAPIImpl.java:72)
	at org.jenkinsci.plugins.gitclient.CliGitAPIImpl$1.execute(CliGitAPIImpl.java:420)
	at org.jenkinsci.plugins.gitclient.CliGitAPIImpl$2.execute(CliGitAPIImpl.java:629)
	at hudson.plugins.git.GitSCM.retrieveChanges(GitSCM.java:1146)
	at hudson.plugins.git.GitSCM.checkout(GitSCM.java:1186)
	at hudson.scm.SCM.checkout(SCM.java:504)
	at hudson.model.AbstractProject.checkout(AbstractProject.java:1208)
	at hudson.model.AbstractBuild$AbstractBuildExecution.defaultCheckout(AbstractBuild.java:574)
	at jenkins.scm.SCMCheckoutStrategy.checkout(SCMCheckoutStrategy.java:86)
	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:499)
	at hudson.model.Run.execute(Run.java:1819)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:97)
	at hudson.model.Executor.run(Executor.java:429)
ERROR: Error cloning remote repo 'origin'
Setting status of 5a72281596876ea79dc81d47ca66a9bde5a9b3f7 to FAILURE with url http://jenkins.leapmind.local:8080
```

Maybe it is full of storage error, please investigate.

**Comment by hadusam** [comment]
run dlk test

**Comment by ruimashita** [comment]
@hadusam 
Thanks 👍
